### DISCUSSION TITLE
Bug in the FactionsX

### DISCUSSION THREAD

**Discussion by H3R0FL000** [ProSavage/FactionsX#13] [Q&A]
Good evening,

I am contacting you because, I am currently co-administrator of a Faction server and we are using your plugin but, at the moment we may have a lot of problems with this one:
The / f map loads one line by one line
The / f upgrade no longer works, we cannot access the different menus of it, but we can duplicate the items that are in the menu.
Some players no longer receive faction invitations, so they can no longer join factions. If we force them into college, they are still considered "Wilderness"

So here are some bugs, if you know how to fix them, I would love to know.

Thanks in advance.

**Comment by prosavage** [comment]
Hi,

Sounds like your data is corrupted, for the upgrade and wilderness thing.

Send the error in console when you click items or open the upgrade menu.

**Comment by prosavage** [comment]
I do not really work on this software anymore, but I am going to try and work updating it in my free time for fun.
